src/routes/dashboard/types.ts

```diff
@@ -0,0 +1,47 @@
+export type Questions = {
```

Comment:
can this file be changed to a type definition file? https://www.typescriptlang.org/docs/handbook/declaration-files/templates/module-d-ts.html